test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.cs

```diff
@@ -21,7 +21,6 @@
 
 namespace Microsoft.Gen.Logging.Test;
 
-[UsesVerify]
```

Comment:
What's the reason for this change?

Reply:
Same question for similar changes in other files.

Reply:
It looks like the change in the Verify's implementations - after upgrading to the latest version I got a compilation error stating there's no such attribute. I removed it and the tests continue to function as expected.